### PR TITLE
Retail autosign grains #16288 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
I hope I got it right.

File: modules/retail/pages/retail-deploy-terminals-auto.adoc [hunk 1]
```diff
@@ -63,7 +81,47 @@ autosign_grains:
 
 
 
-.Procedure: Configuring Saltboot to Auto-Sign During PXE Boot
+// For SLE12 and SLE11 templates, the procedure is the following:
+.Procedure: Configuring Saltboot to Keep Auto-Signed Grains (SLE 11 and SLE 12)
+. In the location where the image source is built, such as a build host or source repository, create a configuration file called [path]``etc/salt/minion.d/autosign-grains.conf``.
+  This must be outside of the root directory provided by the template.
```
Comment (aaannz) [inline modules/retail/pages/retail-deploy-terminals-auto.adoc:87]:
```suggestion
  This must be outside of the [path]``root`` directory provided by the template.
```

Template comes with `root` directory, so we are talking literally about directory named root next to where config.xml is located.

**Review by aaannz — APPROVED** [review]
I have one nitpick to signify `root` directory name, but apart that this is correct. Thanks.

**Comment by keichwa** [comment]
Thank you, Ondrej!

**Pull request merged**